apps/auth0-rules/src/__tests__/on-execute-post-login.test.ts

```diff
@@ -0,0 +1,309 @@
+import { gp2 as gp2Model, UserMetadataResponse } from '@asap-hub/model';
+import { Auth0PostLoginApi } from '@vedicium/auth0-actions-sdk';
+import nock from 'nock';
+
+import { onExecutePostLogin } from '../on-execute-post-login';
+import type {
+  Auth0PostLoginEventWithSecrets,
+  DeepPartial,
+  User,
+} from '../types';
+
+const user: User = {
+  created_at: '2020-08-17T14:01:53.691Z',
+  email: 'joao.tiago@yld.io',
+  email_verified: true,
+  family_name: 'Tiago',
+  given_name: 'Joao',
+  identities: [
+    {
+      provider: 'google-oauth2',
+      access_token: 'accessToken',
+      expires_in: 3599,
+      user_id: 'userId',
+      connection: 'google-oauth2',
+      isSocial: true,
+    },
+  ],
+  locale: 'en',
+  name: 'Joao Tiago',
+  nickname: 'joao.tiago',
+  picture:
+    'https://lh3.googleusercontent.com/a-/AOh14GhcUymAYADTOFTtS7zHdsWbfi2AbCIbpBtxAdNWug',
+  updated_at: '2020-08-21T09:19:39.985Z',
+  user_id: 'google-oauth2|JTsUserId',
+};
+
+const apiUrl = 'http://api.example.com';
+
+const eventBase = {
+  request: {
+    query: {
+      redirect_uri: 'http://example.com',
+    },
+    body: {},
+  },
+  user,
+  secrets: {
+    ASAP_API_URL: apiUrl,
+    AUTH0_SHARED_SECRET: '',
+    PR_APP_DOMAIN: '',
+    AUTH0_ADDITIONAL_CLAIM_DOMAIN: '',
+  },
+} as DeepPartial<Auth0PostLoginEventWithSecrets> as Auth0PostLoginEventWithSecrets;
+
+const apiBase = {
+  access: {
+    deny: jest.fn(),
+  },
+  idToken: {
+    setCustomClaim: jest.fn(),
+  },
+} as Partial<Auth0PostLoginApi> as Auth0PostLoginApi;
+beforeEach(() => {
```

Comment:
I would add a `describe` before
```suggestion
describe('Auth0 Action - Add User Metadata', () => {
beforeEach(() => {
```
the way it was done previously

Reply:
This doesn't make sense when the file already provides this information in the test output